docs/release-notes/release-notes-0.15.2.md

```diff
@@ -0,0 +1,43 @@
+# Release Notes
+
+- [Lightning Terminal](#lightning-terminal)
+    - [Bug Fixes](#bug-fixes)
+    - [Functional Changes/Additions](#functional-changesadditions)
+    - [Technical and Architectural Updates](#technical-and-architectural-updates)
+- [Integrated Binary Updates](#integrated-binary-updates)
+    - [LND](#lnd)
+    - [Loop](#loop)
+    - [Pool](#pool)
+    - [Faraday](#faraday)
+    - [Taproot Assets](#taproot-assets)
+- [Contributors](#contributors-alphabetical-order)
+## Lightning Terminal
+
+### Bug Fixes
+
+### Functional Changes/Additions
+
+* Add support for connecting LiT to the [testnet4
+  network](https://github.com/lightninglabs/lightning-terminal/pull/1120).
+  This can be done using the `--network=testnet4` config option.
+
+### Technical and Architectural Updates
+
+## RPC Updates
+
+## Integrated Binary Updates
+
+### LND
+
+### Loop
+
+### Pool
+
+### Faraday
+
+### Taproot Assets
+
+# Contributors (Alphabetical Order)
+
+* bitcoin-coder-bob
+* Jonathan Harvey-Buschel
```

Comment:
Should JHB be listed here? He hasn't created any PRs towards the next release.

Reply:
him and I worked this issue together in person but I formally typed up the PR. Does that mean he should not be listed?

Reply:
Ah ok, my bad. I missed that mention of him in the PR description. His name should remain.